docs/pages/database-access/introduction.mdx

```diff
@@ -39,14 +39,38 @@ with GitHub, execute a few SQL queries and observe them in the audit log:
   Your browser does not support the video tag.
 </video>
 
-## Getting started
+## Get started
 
 - [Getting started](./getting-started.mdx): Connect Aurora PostgreSQL in a 10
   minute guide.
 
-(!docs/pages/includes/database-access/guides.mdx!)
+## Connect your database
 
-## Resources
+Teleport can protect both self-hosted and managed databases. Learn how to set up
+secure access to databases in your infrastructure:
+
+- Managed [Amazon Web Services](./enroll-aws-databases.mdx) databases
+- Managed [Google Cloud](./enroll-google-cloud-databases.mdx) databases
+- [Self-Hosted](./enroll-self-hosted-databases.mdx) databases, regardless of
+  cloud provider
+- Managed [Microsoft Azure](./enroll-azure-databases.mdx) databases
+- [Database-specific cloud
+  platforms](docs/pages/database-access//enroll-managed-databases.mdx) such as
+  Snowflake and MongoDB Atlas
```

Comment:
I only see cloud-hosted in this list. do we want to add self-hosted link?

Reply:
I addressed this in #38574 and incorporated it when rebasing